packages/react/test/specs/commonTests/handlesAccessibility.tsx

```diff
@@ -133,8 +134,10 @@ export default (
         wrapper.setProps({})
       }
 
-      getEventTargetComponent(component, 'onKeyDown').simulate('keydown', {
-        keyCode: keyboardKey.Enter,
+      act(() => {
```

Comment:
Is this change necessary to be part of this PR? We can improve tests in a separate PR

Reply:
Yep, `Checkbox` uses `useStateManager()` and it performs actions with hooks. Otherwise it produces a React error:

```
  console.error node_modules/react-dom/cjs/react-dom.development.js:506
    Warning: An update to Checkbox inside a test was not wrapped in act(...).
    
    When testing, code that causes React state updates should be wrapped into act(...):
    
    act(() => {
      /* fire events that update state */
    });
    /* assert on the output */
```